kubernetes/windows/setup.ps1

```diff
@@ -31,7 +31,7 @@ Write-Host ('Finished Installing Fluentbit')
 
 Write-Host ('Installing Telegraf');
 try {
-    $telegrafUri='https://dl.influxdata.com/telegraf/releases/telegraf-1.18.0_windows_amd64.zip'
+    $telegrafUri='https://dl.influxdata.com/telegraf/releases/telegraf-1.20.3_windows_amd64.zip'
```

Comment:
same here. i think, we just should cleanup this zip file also.

Reply:
This gets added to /installation
```
Invoke-WebRequest -Uri $telegrafUri -OutFile /installation/telegraf.zip
Expand-Archive -Path /installation/telegraf.zip -Destination /installation/telegraf
```
and then that directory is removed later with `Remove-Item /installation -Recurse` so I think for windows it's ok. I double-checked and didn't see it in the container files